app/controllers/concerns/authentication.rb

```diff
@@ -14,16 +14,18 @@ def authenticate_user!
 
   def login(user)
     reset_session
-    session[:current_user_id] = user.id
+    user.regenerate_session_token
+    session[:current_user_session_token] = user.reload.session_token
```

Comment:
Update the value of `session_token` to prevent a session replay. Use this value to identify the user instead of their id.